stl/inc/cmath

```diff
@@ -1323,6 +1324,19 @@ _NODISCARD constexpr inline long double lerp(
     const long double _ArgA, const long double _ArgB, const long double _ArgT) noexcept {
     return _Common_lerp(_ArgA, _ArgB, _ArgT);
 }
+
+template <class _Ty>
+concept _Arithmetic = is_arithmetic_v<_Ty>;
+
+template <_Arithmetic _Ty1, _Arithmetic _Ty2, _Arithmetic _Ty3>
+_NODISCARD constexpr auto lerp(const _Ty1 _ArgA, const _Ty2 _ArgB, const _Ty3 _ArgT) noexcept {

```

Comment:
Note that there is a comment on line 1312; LWG-3223 says we may not want to implement this. @BillyONeal, do you have any comments?